### ISSUE TITLE
React Native version mismatch 

### ISSUE THREAD

**Issue by ztl8702** [jitsi/jitsi-meet#2979]
I was just hit by an issue that caused mismatched React Native versions when building for Android. https://github.com/facebook/react-native/issues/19259

The workaround is to add this line in the `dependencies` section of `android/app/build.gradle`:

```
	compile("com.facebook.react:react-native:0.51.0") { force = true }
```

According to the discussion on react-native repo, this might be due to someone [updating the react-native version on bintray](https://bintray.com/weimob/maven/react-native#reviews) a few hours ago.

I am lodging this issue because this might affect many people building jitsi-meet for Android. And should the additional line in `gradle.build` be checked-in to prevent similar issues in the future?

**Comment by paweldomas** [comment]
it's fine for me after merging this PR: https://github.com/jitsi/jitsi-meet/pull/2973

**Comment by saghul** [comment]
Fixed in #2973
